SUMMARY.md

```diff
@@ -0,0 +1,20 @@
+# Table of contents
+
+* [Introduction](README.md)
```

Comment:
Should this be `INTRODUCTION.md`?

Reply:
Yes, thanks for noticing. I will try syncing the branch first to check how it looks like before the merging and syncing `master`.